### ISSUE TITLE
SDL client enhancements

### ISSUE THREAD

**Issue by akallabeth** [FreeRDP/FreeRDP#9074]
* full multitouch support (pen, finger, ...)
* clipboard
* multimonitor
* TODO: Additional stuff here

**Comment by amazingfate** [comment]
Will this sdl client be a replacement of mac client? I'm using the current mac client and thinking of implementing multi monitor support on it. But this SDL client seems to be another way.

**Comment by mfleisz** [comment]
@amazingfate yes - SDL client is supposed to replace Mac and Windows clients.

**Comment by akallabeth** [comment]
@amazingfate #8895 is the only blocker for replacing the other platform clients currently. but that could already be worked around by using `SDL3` development branch (just need to implement the clipboard glue code for that)

**Comment by amazingfate** [comment]
> @amazingfate #8895 is the only blocker for replacing the other platform clients currently. but that could already be worked around by using `SDL3` development branch (just need to implement the clipboard glue code for that)

Great work! SDL client has more people involved and I'm happy to try it.
I did test sdl client on from master branch on mac last week, but I saw when running with `/multimon`, these is one monitor displaying black. I'm going to spend some time to investigate it.

**Comment by akallabeth** [comment]
great, multimonitor support is currently very basic and not fully implemented, so any tester and bug fixer is welcome!
